openmdao/core/system.py

```diff
@@ -282,8 +283,8 @@ def pre_setup(self, problem):
         User-configurable method to be run when problem.setup() is called
         but prior to any actual problem setup.
 
-        Args
-        ----
+        Parameters
+        ----------
```

Comment:
FYI, @ mentions work in these reviews, if you want to get someone's attention, @swryan